test/recipes/80-test_ssl_old.t

```diff
@@ -476,18 +476,21 @@ sub testssl {
     subtest 'RSA/(EC)DHE/PSK tests' => sub {
 	######################################################################
 
-	plan tests => 5;
+	plan tests => 6;
 
       SKIP: {
-	  skip "TLSv1.0 is not supported by this OpenSSL build", 5
+	  skip "TLSv1.0 is not supported by this OpenSSL build", 6
 	      if $no_tls1;
 
 	SKIP: {
-	    skip "skipping anonymous DH tests", 1
+	    skip "skipping anonymous DH tests", 2
 	      if ($no_dh);
 
 	    ok(run(test([@ssltest, "-v", "-bio_pair", "-tls1", "-cipher", "ADH", "-dhe1024dsa", "-num", "10", "-f", "-time"])),
 	       'test tlsv1 with 1024bit anonymous DH, multiple handshakes');
+
+	    ok(run(test(['ssltest_old', '-psk', '0102030405', '-cipher', '@SECLEVEL=2:DHE-PSK-AES128-CCM'])),
```

Comment:
This will have the same issue as the other PR.. i.e it needs a seperate skip section

Reply:
I made the change here first, tests break...  Still digging...